frontend/src/components/IstioWizards/FaultInjection/AbortFault.tsx

```diff
@@ -70,7 +71,7 @@ export class AbortFault extends React.Component<Props> {
             />
             <FormHelperText>
               <HelperText>
-                <HelperTextItem>{isValid(this.props.isValid) ? httpStatusMsg : httpStatusMsg}</HelperTextItem>
+                <HelperTextItem>{isValid(this.props.isValid) ? t(httpStatusMsg) : t(httpStatusMsg)}</HelperTextItem>
```

Comment:
This is not your fault, but it doesn't make sense to show the same message for both true and false.
```suggestion
                <HelperTextItem>{t(httpStatusMsg)}</HelperTextItem>
```